cmd/support-logging/res/configuration.toml

```diff
@@ -1,16 +1,31 @@
-Hostname = 'localhost'
-Port = 48061
-Persistence = 'mongodb'
-LoggingFile = './logs/support-logging.log'
-MongoDB = 'logging'
-MongoCollection = 'logEntry'
-MongoURL = 'localhost'
-MongoPort = 27017
-MongoConnectTimeout = 5000
-SocketTimeout = 5000
-MongoUsername = ''
-MongoPassword = ''
+Persistence = 'database'
```

Comment:
I think that mixes the usage of the Databases node in the TOML though. This would then be the only service where Databases["Primary"] could be a file instead of a platform. Additionally, the way it is now, someone can toggle between file and Mongo (for example) by changing the value of only one property if desired.